response_operations_social_ui/views/social/__init__.py

```diff
@@ -15,3 +16,5 @@
 social_bp.add_url_rule('/case/<case_id>/change-response-status',
                        view_func=update_case_response_status, methods=['POST'])
 social_bp.add_url_rule('/iac', view_func=generate_iac, methods=['POST'])
+social_bp.add_url_rule('/response_chasing/<collection_exercise_id>',
```

Comment:
Is it worth making this `/response-chasing` with the hyphen instead of underscore to match with the other routes?